### ISSUE TITLE
Cannot import module

### ISSUE THREAD

**Issue by ivantsov** [recharts/recharts#17]
I've got the following error when tried to import recharts:
```
Module parse failed: D:\...\node_modules\recharts\src\index.js Line 1: Unexpected token
You may need an appropriate loader to handle this file type.
| import 'ima.js-babel6-polyfill';
| import 'core-js/es6/math';
| import Surface from './container/Surface';
```
I think it happens because you have `"main": "src/index.js"` in package.json instead of `"lib/index.js"`.

**Comment by Cloudo** [comment]
Agreed. 
Just mentioned other libraries use `jsnext:main` field in package.json to specify entry point for sources with es6 modules (in our case `src/index.js`), but seems like it can be handled by rollup only. Should check it first...

**Comment by ivantsov** [comment]
> seems like it can be handled by rollup only

Yeah, unfortunately webpack v1 doesn't handle es6 modules :(

> Just mentioned other libraries use jsnext:main field in package.json to specify entry point for sources with es6 modules (in our case src/index.js)

For instance, react-route use it like [this](https://github.com/rackt/react-router/blob/master/package.json#L15). So it would be great to make it the same way.

**Comment by xile611** [comment]
Thanks!
Fixed in v0.5.1.
